src/python/national_voter_file/us_states/nc/transformer.py

```diff
@@ -83,7 +83,7 @@ def extract_do_not_call_status(self, input_dict):
 
     def extract_gender(self, input_dict):
         gender = input_dict['gender_code']
-        if len(gender) == 0:
+        if len(gender) == 0 or gender == ' ':
```

Comment:
Even in circumstances where 'U' is one of the available options? Race and gender and specifically be Unspecified, so it might make sense to set unspecified entries to Unspecified in that instance. Whereas, first name does not have U as an accepted option